requirements/local.txt

```diff
@@ -3,3 +3,4 @@
 -r docs.txt
 
 django-debug-toolbar
+mysqlclient
```

Comment:
Installing `test.txt` already installs `base.txt` which has the `mysqlclient` package from the diff above this file.